core/src/test/java/org/apache/accumulo/core/security/crypto/CryptoTest.java

```diff
@@ -357,14 +363,209 @@ public void testKeyWrapAndUnwrap() throws NoSuchAlgorithmException, NoSuchPaddin
     byte[] wrappedKey = keyWrapCipher.wrap(randKey);
 
     assert (wrappedKey != null);
-    assert (wrappedKey.length == randomKey.length);
+    // AESWrap will produce 24 bytes given 128 bits of key data with a 128-bit KEK
+    assert (wrappedKey.length == randomKey.length + 8);
```

Comment:
I think these are calls are using the java assert statement which by default does nothing.  Java assertions have to be turned on.  Should use the junit assertTrue method.